hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/CalciteSqlOptimizer.java

```diff
@@ -890,8 +902,8 @@ private SqlRowMetadata createRowMetadata(
     private Tuple2<DAG, Set<PlanObjectKey>> createDag(
             PhysicalRel physicalRel,
             QueryParameterMetadata parameterMetadata,
-            Set<PlanObjectKey> usedViews
-    ) {
+            Set<PlanObjectKey> usedViews,
+            @Nullable Map<String, List<Map<String, Expression<?>>>> prunabilities) {
```

Comment:
how many nested generic collections is a code smell?

Reply:
Not aware, sorry.